src/main/markdown/usingeclipse.md

```diff
@@ -35,6 +35,10 @@ GWT release candidates are not bundled with The Google Plugin For Eclipse. If yo
   style="width: 385px; height: 100px;" alt="screenshot"/>
 </div>
 
+A beta version of the community GWT Plugin for Eclipse is available [here](https://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html).  
+
+**DO NOT** attempt to use both versions of the plugin together.
```

Comment:
Seems that the Google Plugin was not updated for Neon, so I think you are right, we should just drop mention of the old Google plugin.